stock_transfer_split_multi/README.rst

```diff
@@ -0,0 +1,10 @@
+Stock Transfer Split Multi
+==========================
+
+This module adds a button *Split multiple units* on the lines of the
+Transfer wizard of the pickings next to the native *Split* button. When
+you click on this button, it will ask you to enter the quantity
+to extract from this line and it will create a new line.
+
+This module has been written by Alexis de Lattre from Akretion
+<alexis.delattre@akretion.com>.
```

Comment:
Here is the template: :wink: 

https://github.com/OCA/maintainers-tools/blob/master/template/module/README.rst